Model/Formatter/Order.php

```diff
@@ -164,4 +165,25 @@ public function isRssAllowed()
         );
     }
 
+    /**
+     * Get order comments in proper format
+     *
+     * @param OrderInterface $orderModel
+     * @return array
+     */
+    public function getOrderComments(OrderInterface $orderModel): array
+    {
+        $comments = [];
+
+        foreach ($orderModel->getStatusHistoryCollection() as $comment) {
+            if ($comment->getIsVisibleOnFront()) {
+                $comments[] = [
+                    'timestamp' => $comment->getCreatedAt(),
+                    'message' => $comment->getComment()
+                ];
+            }
+        }
```

Comment:
So there was a graphql endpoint for invoice comments but resolver was never returning them from default magento?)